app/modelingResource/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.genar.hmssandbox.huawei.modelingresource">
```

Comment:
special name like Genar or Cem should not use as variable or package name in the enterprise libraries